core/src/main/java/org/jboss/wise/core/client/impl/reflection/BasicWSDynamicClientImpl.java

```diff
@@ -225,7 +225,7 @@ public synchronized final void setClassLoader(ClassLoader classLoader) {
 		    //TODO!! Add check on @XmlRegistry
 		    list.add(clazz);
 		} catch (Exception e) {
-		    e.printStackTrace();
+		    //e.printStackTrace();
```

Comment:
Same as above, please log the full exception at least at debug level (otherwise it's lost forever, given the ISE thrown below includes the exception message only.